pyston/resource.py

```diff
@@ -379,7 +379,9 @@ def _deserialize(self):
         if rm in {'POST', 'PUT', 'PATCH'}:
             try:
                 converter = get_converter_from_request(self.request, self.converters, True)
-                self.request.data = self.serializer(self).deserialize(converter.decode(force_text(self.request.body)))
+                self.request.data = self.serializer(self).deserialize(
+                    converter.decode(force_text(self.request.body), resource=self)
+                )
```

Comment:
As resource is passed to converter's encode method (see line 366 in this file), i thought it would be nice to pass it to decode method as well